### DISCUSSION TITLE
初识 Kotlin Flow

### DISCUSSION THREAD

**Discussion by yujinyan** [yujinyan/blog#175] [Comments]
https://blog.yujinyan.me/posts/kotlin-flow-introduction/

**Comment by Elip-kio** [comment]
好像应该是 Kotlin Team `Leader`

**Comment by yujinyan** [comment]
> 好像应该是 Kotlin Team `Leader`

我又查了下好像叫 *Project Lead*，一会儿改下~

https://blog.jetbrains.com/kotlin/2020/11/roman-elizarov-is-the-new-project-lead-for-kotlin/

https://twitter.com/relizarov
